docs_versioned_docs/version-ros2humble/components/yaml/manipulators/universal_robots.mdx

```diff
@@ -0,0 +1,216 @@
+</td>
+</tr><tr>
+<td>
+<center>
+<figure>
+    <img src={require("./img/universal_robots/initialize_4.png").default} width="700" />
+</figure>
+</center>
+</td>
+</tr>
+</table>
+
+Once the robot is in normal mode, you can proceed to initialize the external control program.
+
+
+#### Standard Initialization (Recommended)
+At start up, the robot computer will launch the manipulator's controller using the `clearpath-manipulators.service`. Once the arm is on and initialized through the teach pendant, start the `external_control` program, which will begin communication with the ROS driver.
+
+Start the program by pressing the play button on the **Run** page.
+
+##### Troubleshooting
+If the following error pop-up window appears, then ensure that the arm's URCap configuration matches the robot's computer and that the robot's computer is able to ping the arm. The error also indicates that the ROS driver is not currently running or has been stopped. Restart the `clearpath-manipulators.service` and try again:
+```yaml
+sudo systemctl stop clearpath-manipulators.service
+sudo systemctl start clearpath-manipulators.service
+```
+
+<center>
+<figure>
+    <img src={require("./img/universal_robots/failed.png").default} width="700" />
+</figure>
+</center>
+
+
+#### Headless Initialization
+If the manipulator has been setup to accept external control by surrendering manual control in the teach pendant, then it is possible to bypass the manual initialization of the UR program on the tablet. If this is the case, make sure the arm has been turned on and initialized through the teach pendant, then set, `headless: true` in the `robot.yaml`. This will restart the services and the arm should connect automatically.
+
+##### Setup
+The headless setup is not recommended as it is more complicated, however it exposes the entire teach pendant's functionality to the ROS interface, and thus, provides experienced ROS users with the ability to detect arm faults, clear faults, and restart the arm programmatically.
+
+To begin the setup, make sure that remote control is enabled in the **Settings/System/Remote Control** page:
+
+<center>
+<figure>
+    <img src={require("./img/universal_robots/settings_remote_control.png").default} width="700" />
+</figure>
+</center>
+
+To enter **Remote Control** mode, first put the teach pendant in **Automatic**:
+<center>
+<figure>
+    <img src={require("./img/universal_robots/headless_0.png").default} width="700" />
+</figure>
+</center>
+
+Then, you will be able to switch from **Automatic** to **Remote Control**:
+<center>
+<figure>
+    <img src={require("./img/universal_robots/headless_1.png").default} width="700" />
+</figure>
+</center>
+<center>
+<figure>
+    <img src={require("./img/universal_robots/headless_2.png").default} width="700" />
+</figure>
+</center>
+
+In this mode, you will be unable to interface with the robot directly from the tablet. Instead, all of these commands can be sent directly through the external device.
```

Comment:
Which robot to arm/manipulator to be clear and avoid confuson with the base platform.